packages/app-builder-lib/src/targets/fpm.ts

```diff
@@ -180,10 +180,17 @@ export default class FpmTarget extends Target {
       }
     }
 
-    use(packager.info.metadata.license, it => args.push("--license", it!))
-    use(appInfo.buildNumber, it => args.push("--iteration", it!))
-
-    use(options.fpm, it => args.push(...(it as any)))
+    use(packager.info.metadata.license, it => args.push("--license", it))
+    use(appInfo.buildNumber, it =>
+      args.push(
+        "--iteration",
+        // dashes are not supported for iteration in older versions of fpm
+        // https://github.com/jordansissel/fpm/issues/1833
+        it.split("-").join("_")
```

Comment:
Change to `it.replaceAll('-', '_')`

Reply:
`replaceAll` isn't supported in Node 14.